pkg/platform/openstack/kuryr_bootstrap.go

```diff
@@ -51,6 +53,9 @@ const (
 	MinOctaviaVersionWithTimeouts   = "v2.1"
 	KubernetesEndpointsName         = "kubernetes"
 	KubernetesEndpointsNamespace    = "default"
+	AdmissionControllerCA           = "kuryr-dns-admission-controller-secret"
```

Comment:
This and `WebHookSecret` should probably go into names/names.go and be passed to templates and not hardcoded there.